packages/mdc-textfield/foundation.js

```diff
@@ -57,17 +59,22 @@ class MDCTextFieldFoundation extends MDCFoundation {
       registerBottomLineEventHandler: () => {},
       deregisterBottomLineEventHandler: () => {},
       getNativeInput: () => {},
-      getBottomLineFoundation: () => {},
-      getHelperTextFoundation: () => {},
     });
   }
 
   /**
    * @param {!MDCTextFieldAdapter=} adapter
+   * @param {!FoundationMapType=} foundationMap Map of all subcomponents to subfoundations.
```

Comment:
Nit: "all" here might seem misleading since they're not always required?

Maybe "Map from subcomponents' names to their subfoundations"?